src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -170,3 +171,33 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 		}
 	},
 });
+
+hooks.addPureHook('PATCH', 'resin', 'application', {
```

Comment:
NIT: personally I would prefer to see this hook above the device hooks, following the model in a top-down way